src/api_server/src/lib.rs

```diff
@@ -176,11 +176,7 @@ impl ApiServer {
         }
     }
 
-    pub fn handle_request(
-        &mut self,
-        request: &Request,
-        request_processing_start_us: u64,
-    ) -> Response {
+    fn handle_request(&mut self, request: &Request, request_processing_start_us: u64) -> Response {
```

Comment:
We need this method to be public because we are using it in our fuzzing component that is currently work in progress.

Reply:
updated. Thanks.